mitiq/zne/scaling/folding.py

```diff
@@ -374,7 +374,7 @@ def _create_weight_mask(
 def _create_fold_mask(
     weight_mask: List[float],
     scale_factor: float,
-    folding_method: str,
+    folding_method: Optional[str] = "at_random",
```

Comment:
Since this is a private function, I lean towards removing the argument. Do you think it's important to keep this? Maybe in the situation where we want to add other `folding_method`s in the future?

Reply:
So my only hesitation here is for instance if we wanted to implement something like #2224 , where we might want to fold gates in discrete chunks rather than randomly. If anything, keeping it as an optional argument makes the code more modular.

Reply:
I'm not sure I see how the `_create_fold_mask` function would be used in the LRE implementation, but happy to keep it around at least for a while in case it finds use.

Reply:
@purva-thakre Do you think this would come in handy or would it be better to just remove this argument?

Reply:
@jordandsullivan So far, I don't think we will require `create_fold_mask` for LRE.